packages/medusa/src/services/pricing.ts

```diff
@@ -509,7 +622,169 @@ class PricingService extends TransactionBaseService {
       product.variants.map((productVariant): PricedVariant => {
         const variantPricing = productsVariantsPricingMap.get(product.id)!
         const pricing = variantPricing[productVariant.id]
+
+        Object.assign(productVariant, pricing)
+        return productVariant as unknown as PricedVariant
+      })
+
+      return product
+    })
+  }
+
+  private async getPricingModuleVariantMoneyAmounts(
```

Comment:
question: How will this method change as we add more rule? Right now, we've limited to possible rule_attributes to `region_id`

Reply:
Good question. At the moment we've prioritized to keep the api unchanged as we don't have support for any other mutations of price sets and rules in the admin either, we've included seeders to create the "default" price rules i.e. `region_id` to allow consistency when migrating to the pricing module. Going forward we need a way of handling additional `rule_attribute`/`value` pairs.

I think a way to handle this in the future could be to extend the `MoneyAmount` response with a `rules` attribute and simply add the pairs there when reading, the values of this object would have to have all possible `ruleType`'s for the `PriceSet` in order to properly validate what can be added and what can't.

CC @riqwan would love your take on this as well

Reply:
Agree with that. We have all the needed information here from this query, we just have to change the shape.

Reply:
Sounds good!